calibration/CTDMOG/CGINS-CTDMOG-13605__20170401.csv

```diff
@@ -10,16 +10,16 @@ serial,name,value,notes
 37-13605,CC_i,-2.520471e-004,
 37-13605,CC_j,3.835753e-005,
 37-13605,CC_p_range,1450,
-37-13605,CC_pa0,-9.454889e-004,
-37-13605,CC_pa1,4.515371e-003,
+37-13605,CC_pa0,-9.454890e-04,
+37-13605,CC_pa1,4.515372e-03,
 37-13605,CC_pa2,-3.139491e-011,
 37-13605,CC_ptca0,5.250701e+005,
 37-13605,CC_ptca1,-2.448304e+000,
 37-13605,CC_ptca2,1.064217e-001,
 37-13605,CC_ptcb0,2.530837e+001,
-37-13605,CC_ptcb1,6.750000e-004,
+37-13605,CC_ptcb1,6.750001e-04,
 37-13605,CC_ptcb2,0.00E+00,
 37-13605,CC_ptempa0,1.675834e+002,
 37-13605,CC_ptempa1,-6.420999e-002,
 37-13605,CC_ptempa2,-4.600486e-007,
-37-13605,CC_wbotc,6.0123e-007,
+37-13605,CC_wbotc,6.012297e-07,
```

Comment:
Looks good.

Reply:
Looks good (CD and JL) verified with QCT